python/nvstrings/tests/test_substr.py

```diff
@@ -24,6 +24,13 @@ def test_slice_from():
     assert_eq(got, expected)
 
 
+def test_slice_negative_start():
+    strs = nvstrings.to_device(["a", "b"])
+    got = strs[slice(-3, None, None)]
+    expected = ["a", "b"]
+    assert_eq(got, expected)
```

Comment:
This test should go into a different file, this is for testing elementwise slicing of substrings.

Reply:
I would recommend moving this test to `test_array.py` .  It is closer in function to those methods (gather, sublist).